src/Files.OpenDialog/FilesLauncher/FilesLauncher.cpp

```diff
@@ -1,39 +1,44 @@
-//
+// Copyright (c) 2023 Files
```

Comment:
```suggestion
// Copyright (c) 2023 Files Community
```

Reply:
@yaira2 This is what I’m wondering always. Why Community has being licensed? Do you mean we all have attribution to Files?

Reply:
I’m going to leave it as it is. Just app name.

Reply:
Let @yaira2 decide. I think the copyright notation should be aligned with the LICENSE file.